src/rov/pca9685/external/JHPWMPCA9685/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_library(jhpwmpca9685 ${CMAKE_CURRENT_SOURCE_DIR}/src/JHPWMPCA9685.cpp)
+target_include_directories(jhpwmpca9685 PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(pca9685 PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
Alright so I'm not following, why does Mr. JHGSOIHSDFHS dependency's CMakeLists.txt have a reference to the pca9685 node headers? Or am I misunderstanding the structure?

Reply:
Perhaps this is an error in the original library, but I'm not seeing any references to this name at all in this subdirectory.

Reply:
pca9685 is a library that uses JHPWMPCA9685's headers therefore it needs to know where they are. There is a better, more general way of doing this. Making an environment variable called JHPWMPCA9685_INCLUDE_DIRS is better and allows other libraries/executables to target these include directories without further modification of this file.

Reply:
Fixed in 81db05d8a32334b093dae2fdbe19782b149ac525

Reply:
Okay actually fixed in 05fba948a905919605665b3e8417a0a4eb8ef875, had some cmake scope issues